--- meld-1.4.0/meld/vc/__init__.py	2010-10-16 16:21:54.138033646 +0200
+++ meld-1.4.0/meld/vc/__init__.py.new	2010-10-16 16:22:36.518129763 +0200
@@ -28,7 +28,7 @@
 def load_plugins():
     _vcdir = os.path.dirname(os.path.abspath(__file__))
     ret = []
-    for plugin in glob.glob(os.path.join(_vcdir, "[a-z]*.py")):
+    for plugin in glob.glob(os.path.join(_vcdir, "[a-z]*.pyc")):
         modname = "meld.vc.%s" % os.path.basename(os.path.splitext(plugin)[0])
         ret.append( __import__(modname, globals(), locals(), "*") )
     return ret
