diff -urN meld-3.20.0.orig/data/org.gnome.meld.desktop.in meld-3.20.0/data/org.gnome.meld.desktop.in
--- meld-3.20.0.orig/data/org.gnome.meld.desktop.in	2019-01-20 22:21:14.767559143 +0100
+++ meld-3.20.0/data/org.gnome.meld.desktop.in	2019-01-20 22:21:50.094006006 +0100
@@ -9,6 +9,6 @@
 Terminal=false
 Type=Application
 Icon=org.gnome.meld
-MimeType=application/x-meld-comparison;
+MimeType=application/x-meld-comparison;text/x-patch;
 StartupNotify=true
-Categories=GTK;Development;
+Categories=GTK;GNOME;Development;
