diff -urN meld-0.9.4.1.orig/meld.desktop.in meld-0.9.4.1/meld.desktop.in
--- meld-0.9.4.1.orig/meld.desktop.in	2004-11-16 23:26:59.226328472 +0100
+++ meld-0.9.4.1/meld.desktop.in	2004-11-16 23:34:32.265456056 +0100
@@ -5,8 +5,8 @@
 Exec=meld
 Terminal=false
 Type=Application
-Icon=meld
-Categories=GNOME;Application;Development;
+Icon=meld.png
+Categories=GTK;GNOME;Utility;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=meld
 X-GNOME-Bugzilla-Component=general
