diff -uNr meld-1.6.0.old/data/meld.desktop.in meld-1.6.0/data/meld.desktop.in
--- meld-1.6.0.old/data/meld.desktop.in	2012-04-27 23:09:24.000000000 +0200
+++ meld-1.6.0/data/meld.desktop.in	2012-05-14 13:50:15.306381836 +0200
@@ -9,7 +9,8 @@
 Type=Application
 Icon=meld
 StartupNotify=true
-Categories=GTK;Development;
+Categories=GTK;GNOME;Development;
+MimeType=text/x-patch;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=meld
 X-GNOME-Bugzilla-Component=general
