--- meld-1.8.4/meld/vc/__init__.py.version_control_plugins_glob	2014-04-28 13:12:56.446767163 +0200
+++ meld-1.8.4/meld/vc/__init__.py	2014-04-28 13:13:08.823971551 +0200
@@ -30,7 +30,7 @@ from ._vc import DATA_NAME, DATA_STATE,
 def load_plugins():
     _vcdir = os.path.dirname(os.path.abspath(__file__))
     ret = []
-    for plugin in sorted(glob.glob(os.path.join(_vcdir, "[a-z]*.py"))):
+    for plugin in sorted(glob.glob(os.path.join(_vcdir, "[a-z]*.pyc"))):
         modname = "meld.vc.%s" % os.path.basename(os.path.splitext(plugin)[0])
         ret.append( __import__(modname, globals(), locals(), "*") )
     return ret
