--- meld-3.24.0/data/org.gnome.Meld.desktop.in.in.orig	2026-07-04 19:31:45.347039418 +0200
+++ meld-3.24.0/data/org.gnome.Meld.desktop.in.in	2026-07-04 19:32:05.630262867 +0200
@@ -9,6 +9,6 @@ Terminal=false
 Type=Application
 # TRANSLATORS: Do NOT translate or transliterate this text (this is an icon file name)!
 Icon=@icon@
-MimeType=application/x-meld-comparison;
+MimeType=application/x-meld-comparison;text/x-patch;
 StartupNotify=true
-Categories=GTK;Development;
+Categories=GTK;GNOME;Development;
