--- meld-1.3.1/vc/__init__.py	2009-08-14 07:01:33.000000000 +0200
+++ meld-1.3.1/vc/__init__.py.new	2009-09-30 20:18:47.250060327 +0200
@@ -28,7 +28,7 @@
 def load_plugins():
     _vcdir = os.path.dirname(os.path.abspath(__file__))
     ret = []
-    for plugin in glob.glob(os.path.join(_vcdir, "[a-z]*.py")):
+    for plugin in glob.glob(os.path.join(_vcdir, "[a-z]*.pyc")):
         modname = "vc.%s" % os.path.basename(os.path.splitext(plugin)[0])
         ret.append( __import__(modname, globals(), locals(), "*") )
     return ret
