--- meld-3.20.1/meld/build_helpers.py.orig	2020-01-04 09:01:35.032939451 +0100
+++ meld-3.20.1/meld/build_helpers.py	2020-01-06 12:55:45.200685245 +0100
@@ -66,10 +66,6 @@
 
 
 class MeldDistribution(distutils.dist.Distribution):
-    global_options = distutils.dist.Distribution.global_options + [
-        ("no-update-icon-cache", None, "Don't run gtk-update-icon-cache"),
-        ("no-compile-schemas", None, "Don't compile gsettings schemas"),
-    ]
 
     def __init__(self, *args, **kwargs):
         self.no_update_icon_cache = False
@@ -407,17 +407,26 @@
 
 
 class install_data(distutils.command.install_data.install_data):
+    user_options = distutils.command.install_data.install_data.user_options + [
+        ("no-update-icon-cache", None, "Don't run gtk-update-icon-cache"),
+        ("no-compile-schemas", None, "Don't compile gsettings schemas"),
+    ]
+
+    def initialize_options(self):
+        distutils.command.install_data.install_data.initialize_options(self)
+        self.no_update_icon_cache = None
+        self.no_compile_schemas = None
 
     def run(self):
         distutils.command.install_data.install_data.run(self)
 
-        if not self.distribution.no_update_icon_cache:
+        if not self.no_update_icon_cache:
             # TODO: Generalise to non-hicolor icon themes
             info("running gtk-update-icon-cache")
             icon_path = os.path.join(self.install_dir, "share/icons/hicolor")
             self.spawn(["gtk-update-icon-cache", "-q", "-t", icon_path])
 
-        if not self.distribution.no_compile_schemas:
+        if not self.no_compile_schemas:
             info("compiling gsettings schemas")
             gschema_path = build_data.gschemas[0][0]
             gschema_install = os.path.join(self.install_dir, gschema_path)
