--- meld-1.1.3/help/C/GNUmakefile.orig	2006-02-01 22:54:56.000000000 +0100
+++ meld-1.1.3/help/C/GNUmakefile	2006-02-01 22:55:15.000000000 +0100
@@ -10,7 +10,7 @@
 
 .PHONY : all
 all $(OMF_NAME).install : $(OMF_NAME)
-	scrollkeeper-preinstall $(INST_XML) $(OMF_NAME) $(OMF_NAME).install
+	scrollkeeper-preinstall $(XML_DIR_)/meld.xml $(OMF_NAME) $(OMF_NAME).install
 
 .PHONY : install
 install : $(OMF_NAME).install
@@ -18,7 +18,7 @@
 	install -m 644 meld.xml $(XML_DIR_)/meld.xml
 	install -m 644 figures/*.png $(XML_DIR_)/figures
 	-install -m 644 $< $(OMF_DIR_)/$(OMF_NAME)
-	-scrollkeeper-update -p $(OMF_STATE) -o $(OMF_DIR_)
+	-scrollkeeper-update -o $(OMF_DIR_)
 
 .PHONY : uninstall
 uninstall :
--- meld-1.1.3/help/es/GNUmakefile.orig	2006-02-01 22:54:07.000000000 +0100
+++ meld-1.1.3/help/es/GNUmakefile	2006-02-01 22:54:48.000000000 +0100
@@ -10,7 +10,7 @@
 
 .PHONY : all
 all $(OMF_NAME).install : $(OMF_NAME)
-	scrollkeeper-preinstall $(INST_XML) $(OMF_NAME) $(OMF_NAME).install
+	scrollkeeper-preinstall $(XML_DIR_)/meld.xml $(OMF_NAME) $(OMF_NAME).install
 
 .PHONY : install
 install : $(OMF_NAME).install
@@ -18,7 +18,7 @@
 	install -m 644 meld.xml $(XML_DIR_)/meld.xml
 	install -m 644 figures/*.png $(XML_DIR_)/figures
 	-install -m 644 $< $(OMF_DIR_)/$(OMF_NAME)
-	-scrollkeeper-update -p $(OMF_STATE) -o $(OMF_DIR_)
+	-scrollkeeper-update -o $(OMF_DIR_)
 
 .PHONY : uninstall
 uninstall :
