--- meld-3.22.0/meld/build_helpers.py.orig	2022-09-06 21:30:17.648921101 +0200
+++ meld-3.22.0/meld/build_helpers.py	2022-09-07 06:20:41.447497325 +0200
@@ -64,7 +64,7 @@ def has_data(self):
     return "build_data" in self.distribution.cmdclass
 
 
-command_base.command.build.Build.sub_commands.extend([
+command_base.command.build.build.sub_commands.extend([
     ("build_i18n", has_i18n),
     ("build_icons", has_icons),
     ("build_help", has_help),
