diff -urN aa/meld-0.9.5/help/C/GNUmakefile meld-0.9.5/help/C/GNUmakefile
--- aa/meld-0.9.5/help/C/GNUmakefile	2004-10-28 20:24:03.000000000 +0200
+++ meld-0.9.5/help/C/GNUmakefile	2005-02-03 21:35:46.768797744 +0100
@@ -10,7 +10,7 @@
 
 .PHONY : all
 all $(OMF_NAME).install : $(OMF_NAME)
-	scrollkeeper-preinstall $(INST_XML) $(OMF_NAME) $(OMF_NAME).install
+	scrollkeeper-preinstall file:$(XML_DIR_)/meld.xml $(OMF_NAME) $(OMF_NAME).install
 
 .PHONY : install
 install : $(OMF_NAME).install
